src/context-selection/section-filter-selector-bar-item/section-filter-selector-bar-item.js

```diff
@@ -55,9 +54,7 @@ export default function SectionFilterSelectorBarItem() {
 
     const selectableOptions = dataSet?.renderAsTabs
         ? sectionOptions
-        : [{ value: undefined, label: i18n.t('All sections') }].concat(
-              ...sectionOptions
-          )
+        : [...sectionOptions, { value: null, label: i18n.t('All sections') }]
```

Comment:
Here you're having null as value, while checking strictly for undefined above. Is that intentional?

Reply:
Ah yes, i decided to take advantage of the philosophical differentiation between underlined and null, where null in this case is an intentional absence, where undefined is not set yet. Does it make sense to you?